main_loop_tf/main.py

```diff
@@ -730,19 +843,39 @@ def main_loop(placeholders, val_placeholders, train_outs, train_summary_op,
             # TODO: It should be IoU or any other metric, but in this
             # case our loss is Dice Coefficient so it's fine
             loss_value = -1.0 if loss_value < -cfg.thresh_loss else loss_value
-            in_values = [x_in, y_in, split_dim, labels_split_dim,
-                         1 + loss_value]
-            feed_dict = {p: v for (p, v) in zip(placeholders, in_values)}
+            in_values = x_batch_chunks + y_batch_chunks + [loss_value]
+            if x_batch.shape[0] != dataset_params['batch_size']:
+                s_placeholders = (placeholders[0][:s_gpus['train']] +
+                                  placeholders[1][:s_gpus['train']] +
+                                  [placeholders[2]])
+                feed_dict = {p: v for(p, v) in zip(s_placeholders,
+                                                   in_values)}
```

Comment:
MINOR: This can be moved inside the previous if/else on the same condition to keep similar things together